src/analytics.coffee

```diff
@@ -6,8 +6,10 @@ define [
 ], (Settings, Session, PluginsManager, ActionsManager) ->
   class Analytics
     constructor: ->
-      new Session().run().then (session) =>
-        plugins_manager = new PluginsManager(session)
+      plugins_manager = new PluginsManager
+
+      new Session(plugins_manager).run().then (session) =>
+        plugins_manager.session = session
```

Comment:
Do we actually need this one?
We are already passing a reference inside `connect` command.